apps/test/unit/lib/ui/SettingsCogTest.js

```diff
@@ -68,26 +74,27 @@ describe('SettingsCog', () => {
     }, 0);
   });
 
-  it('only shows manage assets when maker toggle is false', () => {
+  it('does not show maker toggle when "showMakerToggle" is false', () => {
     const wrapper = mount(<SettingsCog showMakerToggle={false} />);
-    const cog = wrapper.find(FontAwesome).first();
-    cog.simulate('click');
-    const menuWrapper = getPortalContent(wrapper);
-    const numMenuItems = menuWrapper.find(PopUpMenu.Item).length;
-    expect(numMenuItems).to.equal(1);
+    expect(wrapper.find(ToggleMaker)).to.have.lengthOf(0);
+    // wrapper.instance().open();
```

Comment:
Is there a reason to keep these comments?

Reply:
Woops. Nope. Thanks for catching that!